<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-AOC2RQRL" failures="1" tests="2" name="CucumberRunner.Testngrunner" time="42.531" errors="0" timestamp="2023-01-13T15:28:53 IST" skipped="0">
  <testcase classname="CucumberRunner.Testngrunner" name="runScenario" time="17.793">
    <failure message="test is fail expected [Beans] but found [Bean]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: test is fail expected [Beans] but found [Bean]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at Cucumberstepdefinatons.Checkoutpagestepdefinations.user_is_proceeds_to_the_cheack_validate_the_productname_in_the_checkout_page(Checkoutpagestepdefinations.java:35)
at âœ½.User is proceeds to the cheack validate the productname Bean in the checkout page(file:///C:/Users/Kunal/eclipse-workspace/Greenkart/src/test/java/CucumberFeature/Cheackout.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="CucumberRunner.Testngrunner" name="runScenario" time="24.738"/>
  <system-out/>
</testsuite> <!-- CucumberRunner.Testngrunner -->
